internal/pkg/workspace/errors.go

```diff
@@ -65,3 +73,12 @@ func (e *errHasExistingApplication) Error() string {
 	}
 	return fmt.Sprintf("workspace is already registered with application %s under %s", e.existingAppName, relPath)
 }
+
+// IsEmptyErr returns true if the error is related to an empty workspace.
+func IsEmptyErr(err error) bool {
+	type empty interface {
+		empty() bool
+	}
```

Comment:
the only thing i wonder - it's kinda like "is empty if the function exists", so maybe it could just be `empty()` (no bool)?

also, prob don't need to define a type for `empty`, just:
```
var emptyWs interface {
  empty()
}
```

Reply:
Done!